opensrp-chw-core/src/main/java/org/smartregister/chw/core/utils/ChwNotificationUtil.java

```diff
@@ -0,0 +1,72 @@
+package org.smartregister.chw.core.utils;
+
+import android.content.Context;
+
+import org.json.JSONArray;
+import org.json.JSONTokener;
+import org.smartregister.chw.core.R;
+import org.smartregister.clientandeventmodel.Event;
+import org.smartregister.util.JsonFormUtils;
+
+import java.util.Date;
+import java.util.HashMap;
+import java.util.Map;
+
+import timber.log.Timber;
+
+public class ChwNotificationUtil {
+
+    public static String getNotificationDetailsTable(Context context, String notificationType) {
+        if (context.getString(R.string.notification_type_sick_child_follow_up).equals(notificationType)) {
+            return "ec_sick_child_followup";
+        } else if (context.getString(R.string.notification_type_pnc_danger_signs).equals(notificationType)) {
+            return "ec_pnc_danger_signs_outcome";
+        } else if (context.getString(R.string.notification_type_anc_danger_signs).equals(notificationType)) {
+            return "ec_anc_danger_signs_outcome";
+        } else if (context.getString(R.string.notification_type_malaria_follow_up).equals(notificationType)) {
+            return "ec_malaria_followup_hf";
+        } else if (context.getString(R.string.notification_type_family_planning).equals(notificationType)) {
+            return "ec_family_planning_update";
+        }
+        return null;
+    }
```

Comment:
Can this be converted to switch statement @allan-allay

Reply:
@ellykits Yeah a `switch` would have been cleaner but it isn't possible to have the string resources we're referencing in a switch statement because they're not constant expressions. 

> R.id.*, since ADT 14 are not more declared as final static int so you can not use in switch case construct.